pages/regions/cancel_deployment_modal.py

```diff
@@ -0,0 +1,43 @@
+#
+# $Id$
+#
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class CancelDeploymentModalPage(Base):
+    """Cancel Deployment Modal Window:
+    1) Modal Header
```

Comment:
Some of these steps don't make sense to me...they don't seem like steps the way they are worded.   I'm confused which is not to hard to do.